Upstream: not yet
Reason: fails if python3 is system python provider

From f4607ed246476b3ed58b4ce25cfdc6348a4a6bd1 Mon Sep 17 00:00:00 2001
From: Kylie McClain <somasis@exherbo.org>
Date: Thu, 27 Aug 2015 21:29:48 -0400
Subject: [PATCH] utils/key2pub: use python2 explicitly

---
 utils/key2pub.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/key2pub.py b/utils/key2pub.py
index 3e84cd2..8bea042 100755
--- a/utils/key2pub.py
+++ b/utils/key2pub.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import sys
 try:
-- 
2.5.0

